### PR TITLE
Add resource reads for ticket fields

### PR DESCRIPTION
**Pull request by tamccall**

### CODE REVIEW

**Comment by tamccall** [comment]
This is a work in progress will require https://github.com/nukosuke/go-zendesk/pull/50 to be merged before it can be finished

**Comment by nukosuke** [comment]
Merged https://github.com/nukosuke/go-zendesk/pull/50 :pray:

File: zendesk/resource_zendesk_ticket_field.go [hunk 1]
```diff
@@ -184,7 +190,59 @@ func resourceZendeskTicketFieldCreate(d *schema.ResourceData, meta interface{})
 	return resourceZendeskTicketFieldRead(d, meta)
 }
 
+func setSchemaFields(d *schema.ResourceData, m map[string]interface{}) error {
```
Comment (nukosuke) [inline zendesk/resource_zendesk_ticket_field.go]: I think this function is useful for other resources. Please move this to utility.go or something like that.

**Comment by nukosuke** [comment]
I confirmed.
Thank you!

**Pull request merged**